shanoir-ng-studies/src/main/java/org/shanoir/ng/subjectstudy/model/SubjectStudyStudyCardTag.java

```diff
@@ -0,0 +1,61 @@
+package org.shanoir.ng.subjectstudy.model;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import org.shanoir.ng.tag.model.Tag;
+
+@Entity
+@Table
+public class SubjectStudyStudyCardTag implements Serializable {
+	
+	private static final long serialVersionUID = -7148798110331644158L;
+	
+	@Id
+    @Column(name = "subject_study_id")
+    private Long id;
+	
+	@OneToOne
+    @JoinColumn(name = "subject_study_id")
+    private SubjectStudy subjectStudy;
+	
+	private int type;
+
+	public int getType() {
+		return type;
+	}
+
+	public void setType(int type) {
+		this.type = type;
+	}
+	
+	public Tag getTag() {
+		Tag tag = new Tag();
+		if (type == 1) {
```

Comment:
Add some constants ? (and use them further in code ?)

Reply:
not sure, what a constant might change here as there is no other usage in code and the entire method is only a work-around to change the GUI display according to the type, to have a display beta version without spending to much time on it